examples/github/github-eventlistener-interceptor.yaml

```diff
@@ -14,6 +14,11 @@ spec:
               secretKey: secretToken
             eventTypes:
               - pull_request
+        - cel:
+            filter: >-
+              body.action == 'opened' ||
```

Comment:
You should be able to do `body.action in ['opened', 'synchronize','reopened']`

Reply:
Cleaner, thanks!